docs/design/policy-engine-openapi.yaml

```diff
@@ -9,6 +9,30 @@ info:
 servers: []
 paths:
   "/graph":
+    parameters: &parameters
```

Comment:
This is currently missing the `channel` that you see in [your test results][1], although I'm not clear on how that's possible.  Is this YAML content not covered by CI?

[1]: https://github.com/openshift/cincinnati/pull/1031#issuecomment-3080992997